weather_getter/app.py

```
@@ -2,22 +2,56 @@
 from datetime import datetime
 from geopy.geocoders import Nominatim
 import json
+import sys
+import re
 
 
 city_name = input("Please Enter City Name:")
 
+if re.fullmatch(r'^[a-zA-Z\s]+$',city_name) == None:
+    print("The city name is invalid.")
+    sys.exit()
+
```

Try to raise exception instead of printing and exiting.